interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -91,6 +95,7 @@ public static void main(String[] args) throws Exception {
   private int soakPerIterationMaxAcceptableLatencyMs = 1000;
   private int soakOverallTimeoutSeconds =
       soakIterations * soakPerIterationMaxAcceptableLatencyMs / 1000;
+  private static LoadBalancerProvider customBackendMetricsLoadBalancerProvider;
```

Comment:
nit: Don't bother deregistering. We control main, so aren't concerned about any state left over after it returns.